### PR TITLE
refactor: cleanup

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**

### CODE REVIEW

File: .github/workflows/documentationjs.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Deploy documentation.js on GitHub pages
```
Comment (targos) [inline .github/workflows/documentationjs.yml:1]: Sorry I forgot to mention in the issue that we should not add a documentationjs workflow if there is already a typedoc workflow

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -13,4 +13,4 @@ jobs:
       npm: true
     secrets:
       github-token: ${{ secrets.BOT_TOKEN }}
-      npm-token: ${{ secrets.NPM_BOT_TOKEN }}
+      npm-token: ${{ secrets.NPM_BOT_TOKEN }}
```
Comment (targos) [inline .github/workflows/release.yml:16]: The new line is missing again :(

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -14,3 +14,4 @@ jobs:
     secrets:
       github-token: ${{ secrets.BOT_TOKEN }}
       npm-token: ${{ secrets.NPM_BOT_TOKEN }}
+
```
Comment (targos) [inline .github/workflows/release.yml:17]:
```suggestion

```

Again, there should be a new line, but no space.

File: .github/workflows/release.yml (hunk 2 above)
Comment (wadjih-bencheikh18) [reply .github/workflows/release.yml]: fixed , sorry about that

File: .github/workflows/release.yml (hunk 3 above)
Comment (wadjih-bencheikh18) [reply .github/workflows/release.yml]: okey ,done

**Pull request merged**